### PR TITLE
Adjust time - Updated test and refactoring

### LINKED ISSUE #36: refactor adjust.time.R
- [x] Implement a test case for adjust.time.R
- [ ] Refactor the code by extracting functions and using `dplyr` and some tidy code over working with native dataframes
- [ ] Remove not needed inputs and outputs

### LINKED ISSUE #56: Implement tests with proper test data for `adjust.time` method
The test data is available here: https://gitlab.ics.muni.cz/umsa/umsa-files/-/tree/aplcms_testdata/testdata/recetox-aplcms

### PR DESCRIPTION
**Pull request by zargham-ahmad**
This pr contains updated test (into parameterized form) and test data which closes #56.
It also contains some more refactoring for adjust time function which close #36 and also I have reused the graph function which Matej extracted in pr #63, which cover adjust time part of issue #61. 
I also noticed during refactoring that even in case of else block of 'only one sample' execution the function was still returning corrected rt time which don't exist in this case, so I have also added a check for that.

### CODE REVIEW

**Comment by hechth** [comment]
Is there an issue related to this PR? If so, please link it using one of the [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

File: tests/testthat/test-adjust-time.R [hunk 1]
```diff
@@ -1,48 +1,61 @@
...
-  expected <- lapply(expected, as.matrix)
-
-  corrected <- adjust.time(
-    features = extracted,
-    mz.tol = NA,
-    chr.tol = NA,
-    find.tol.max.d = 10 * 1e-05,
-    max.align.mz.diff = 0.01,
-    do.plot = FALSE
+    expect_equal(corrected, expected)
```
Comment (xtrojak) [inline tests/testthat/test-adjust-time.R:48]: So here we are comparing the files as they are, with no tolerances etc. Compared to tests of `extract features`, this is less robust. @hechth do you think this is enough in this case?

**Comment by zargham-ahmad** [comment]
@hechth Yes the issues are already linked but the closing keywords were not used, I have updated the description with the closing keywords.

File: tests/testthat/test-adjust-time.R (hunk 1 above)
Comment (hechth) [reply tests/testthat/test-adjust-time.R:48]: Yeah - there seems to be no problem related to probrabilities etc. in this part, it seems to be only in the extract features part so far.

**Pull request merged**